Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/states/TracingState.kt

```diff
@@ -114,12 +115,39 @@ data class LowRisk(
         )
     }
 
+    fun getRiskContactBodyDescription(c: Context): String = if (daysWithEncounters == 0) {
```

Comment:
Please rename c to context

Reply:
Renamed all parameters in this class from `c` to `context`